content/blog/integrating-istio-and-spire.md

```diff
@@ -0,0 +1,463 @@
+
+  Now simply patch the ingress-gateway with spiffe.io/spire managed-identity: true label.
+
+   This will register your ingress-gateway pod into the server.  
+
+  ```shellsession
+  kubectl patch deployment istio-ingressgateway -n istio-system -p '{"spec":{"template":{"metadata":{"labels":{"spiffe.io/spire-managed-identity": "true"}}}}}'
+  ```
+
+  After patching, confirm the working of your ingress-gateway pod, istiod, and all their containers.
+
+## Step 4: Deploying Sample Application
+
+Now that our SPIRE and Istio are integrated, the identities to workloads must be issued by SPIRE. 
+
+For our case, we will create a namespace “bookinfo” and will add a label **“spiffe.io/spire-managed-identity: true”** to it, then we will create a new ClusterSPIFFEID CRD with **namespace selector** with match label as “spiffe.io/spire-managed-identity: true.” 
+
+Now when the new workload is added to this namespace or any other namespace which has the label mentioned above, then automatically it will get registered in the server. 
+
+**4.1** Create a new namespace.    
+
+```shellsession
+kubectl create namespace <insert-namespace-name-here>
+```
+
+**4.2** Add a label to it, same as that you have used for clusterSPIFFEID.
+
+```shellsession
+kubectl label namespaces <namespace_name> spiffe.io/spire-managed-identity=true
+```
+
+**4.3** Enable istio-injection for this namespace so that any new pods that are created in that namespace will automatically have a sidecar added to them. You can achieve this by just adding another label in similar fashion.
+
+```shellsession
+kubectl label namespace <namespace_name> istio-injection=enabled --overwrite
+```
+
+After all edits to your namespace, the namespace would look similar as shown below.
+
+**Note:** You can edit further if you want using following command, but take care that your resulting yaml is not invalid. You can validate your yaml using any online validator available.
+
+```shellsession
+kubectl edit ns <namespace_name>
+```
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"v1","kind":"Namespace","metadata":{"annotations":{},"labels":{"name":"backend"},"name":"bookinfo"}}
+  creationTimestamp: "2023-03-21T06:23:38Z"
+  labels:
+    istio-injection: enabled
+    kubernetes.io/metadata.name: bookinfo
+    spiffe.io/spire-managed-identity: "true"
+  name: bookinfo
+  resourceVersion: "6116523"
+  uid: 6193a0b7-8455-46bd-a456-797ef69c045a
+spec:
+  finalizers:
+  - kubernetes
+status:
+  phase: Active
+```
+
+**4.4** Create and apply a ClusterSPIFFEID CRD with namespace selector. 
+
+Copy the clusterSPIFFEID from **[this link](https://raw.githubusercontent.com/cxteamtrials/caas-trials-content/main/services/spire/clusterspiffeid-example.yaml)** and just change the selector to **namespace selector** and make sure that the correct match label is there like shown below. 
```

Comment:
Break this out into two sentences. End the first sentence with "...**namespace selector**." Make the next sentence read as follows:  "Make sure that the correct match label is there, as shown below:"